app/jsx/App.jsx

```diff
@@ -13,6 +13,11 @@ import { BrowserRouter, Route, Switch } from 'react-router-dom'
 import ReactGA from 'react-ga'
 import ReactModal from 'react-modal'
 
+// Import klaroConfig
+import klaroConfig from './klaro-config.jsx';
+// Log klaroConfig to the console
+console.log(klaroConfig);
```

Comment:
Do you want to delete this?

Reply:
I kind of like leaving these in, as proof to Klaro skeptics.

Reply:
Oh, though... this is the entire klaroConfig... that's not necessary, I'll take that one out